docs/walkthroughs/connecting-device-to-service.adoc

```diff
@@ -0,0 +1,48 @@
+[[mobile-core-public-ip]]
+= Mobile Core installation with public IP
+To start openshift cluster for local development and connect your device to same network you need to specify public IP of
+your laptop or server you want openshift being installed and provide following parameters
+```
+Use public IP yes/no?: _yes_
+Cluster IP local/public: <your-eth0-ip>
+```
+image::images/mobile-core-public-ip-setup.png[openshift-mobile-core-public-ip-setup]
+Find IP address of your local machine you can with this command ```ifconfig | grep inet```.
```

Comment:
Find IP address of your local machine you can with this command > To Find the IP address of your local machine use the following command